.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Report a bug
+about: Report a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Thank you for reporting this issue! -->
+
+Let's make sure issue is not already fixed in latest builds first.
+
+- [ ] I verified I can reproduce this issue against [latest I-Build of Eclipse SDK](https://download.eclipse.org/eclipse/downloads/)
```

Comment:
While I understand the idea of asking for a reproduction with latest I-Build, it could prevent people from posting the issue, as they do not understand what an I-Build is and how to set it up (eben though it's equal to setting up a released IDE) or as they may as least not want to invest the effort.
So I would at least suggest to make this verification step kind of optional and clarify that in the template.

Reply:
> as they do not understand what an I-Build is and how to set it up 

Is it clearer if we say "a recent build" with the same link?

> as they may as least not want to invest the effort.

If there issue is not even worth for them to verify it, it's most likely not worth for us to worry about it ;)
I really believe that people should understand as users/reporters that they are the ones who can first put a higher bar on project quality, just by verifying their issues and improving their reports.

Reply:
We can start the template with a note like
```
<!-- Thank you for reporting this issue! -->
<!-- The following template is highly recommended to trigger an efficient course of actions: it's pretty suitable if you're not familiar with the project as it will guide you in sharing the most important information. If you're not comfortable with this template for your particular report, feel free to erase it and write your report from scratch -->
```
what do you think?

Reply:
That sounds good 👍 Thanks!

Reply:
Can we still have "I-Build" replaced with "Integration Build" to match the terminology on the web page for those who are not familiar with the Eclipse terminology?